src/modules/users/components/user-list.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {FC, useState} from 'react';
+import {User} from "../user-types";
+
+export const UserList: FC = () => {
+
+    const [users, setUsers] = useState<User[]>([])
+
+    const addUser = (firstName: String, lastName: String) => {
+        const userToAdd : User = {
+            userName: {
+                firstName: firstName,
+                lastName: lastName
+            },
+            id: users.length
+        };
+
+        setUsers(users.concat([userToAdd]));
```

Comment:
Může být, ale častěji se používá:

```suggestion
        setUsers([...users, userToAdd]);
```